app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.samplekotlin">
```

Comment:
Is removing `package` backward compatible with older versions?

Reply:
Nope, but I understand that we target the version we want in our sample app and we keep it consistent with the required interfaces.

Also, if somebody downgrades the sample app, they will get a nice error message saying that you need this in your manifest. This is not specially obscure to patch if you force it to happen.